source/includes/get-started/troubleshoot.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   If you run into issues on this step, ask for help in the
+   :community-forum:`MongoDB Community Forums <tag/ruby/>`
+   or submit feedback by using the :guilabel:`Rate this page`
+   tab on the right or bottom right side of this page.
```

Comment:
Q: do you know why we specify both locations? this was only on the top right on the pages i checked

Reply:
Maybe it depends on the browser/device? not sure, but I'll reword to just "right side"